### PR TITLE
Fix some build error

### PR DESCRIPTION
**Pull request by ramesses2nd**
Fix some build errors
- add dependency for image to cmake file
- change class name
- add chrono namespace
- declare `message` variable

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -26,16 +26,19 @@ if(BUILD_TESTING)
...
+  DESTINATION lib/${PROJECT_NAME})
 
-add_executable(controller src/controller.cpp)
-ament_target_dependencies(controller rclcpp geometry_msgs)
-install(TARGETS controller
-	DESTINATION lib/${PROJECT_NAME})
+#add_executable(controller src/controller.cpp)
+#ament_target_dependencies(controller rclcpp geometry_msgs)
+#install(TARGETS controller
+#	DESTINATION lib/${PROJECT_NAME})
```
Comment (poifull10) [inline CMakeLists.txt]: Could you remove these comments? (this problems are solved in master)

File: src/image_publisher.cpp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 int main(int argc, char** argv)
 {
   rclcpp::init(argc, argv);
-  auto node = std::make_shared<ros2_mock::ControllerNode>();
+  auto node = std::make_shared<ros2_mock::ImagePublisher>();
```
Comment (poifull10) [inline src/image_publisher.cpp]:
```suggestion
  const auto node = std::make_shared<ros2_mock::ImagePublisher>();
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -26,16 +26,19 @@ if(BUILD_TESTING)
...
 endif()
 
-add_executable(sensor src/sensor.cpp)
-ament_target_dependencies(sensor rclcpp geometry_msgs)
-install(TARGETS sensor
-  DESTINATION lib/${PROJECT_NAME})
+#add_executable(sensor src/sensor.cpp)
+#ament_target_dependencies(sensor rclcpp geometry_msgs)
+#install(TARGETS sensor
+#  DESTINATION lib/${PROJECT_NAME})
```
Comment (poifull10) [inline CMakeLists.txt]: please remove #s

File: CMakeLists.txt (hunk 1 above)
Comment (ramesses2nd) [reply CMakeLists.txt]: I removed these line

File: CMakeLists.txt (hunk 3 above)
Comment (ramesses2nd) [reply CMakeLists.txt]: I remove these lines

**Comment by ramesses2nd** [comment]
Hi poifull10,

I removed the comment lines.
Could you check my commit again?

**Comment by ramesses2nd** [comment]
I revert the latest commit.

**Comment by poifull10** [comment]
LGTM, good works! :+1:

**Pull request merged**